src/assets/svg/avatar.svg

```diff
@@ -5,6 +5,7 @@
cMsdxk4WcFitmuq4ccRNpB+vbdlzD9nzVuY3We39Rqo5o2PjcHNcKgjeyVjmuaHAihBFQQn4XinwwLos3MzLP/R8QH40mBdQ1dETdvbUKOZjXseC07wV7rixWFZ5M5GD8Oo0+D25GbCvrG76tORUE9ATwP7H9DvqhgpRPC3+DIch+B3b6dvATkE852QbyuKw9A2QlvZ1wm2EkVDoVhjT5/QLA4mGSKTicx4oRRSl7g2RvDWxOZCp803oFCzoT9SmtFA0DmZIjY2QNLoGl0CgebINQnMIBKBpdVpdB3OOiOiyuq0vzpYUbIkDxf//EACIRAAMAAQQDAQEBAQAAAAAAAAECAwAEBQYSBxETCBAUFf/aAAgBAwEBAgA3VYbTtPj/AGrxlt2zRhBQgn8viYGB0/8AnOn/AM/w+Hw+PyMjNpmZmZlNL4y0Pjfb+LDSLpE0qRWagKE+XxMTAwMDD4GHxMjIzM2k02mZmZ2v/nHRHTf5xEJ6AAUKAny+RkZGRkZGRkZmTSabTM2mZ0m4LEsD/AQwZSpXFwL8zMzMzMoUabTMzNpmbSMqrWdFLfQuSW7ii1SqVSiOCQQVKlSpBUoZmZkYtKoplMqj4X+hcnuKradkslkqHw/wjCChXp8jJkZKiqUVy5cE9u5YsHWs7TtOqUVsIIK+uvQSM2Qy/wA52a/FdZw7X7NRXxsJ7die3ZazrO06zorA+ivXosvkZtOl727PRrNqNw2jc+L0RsOduxIb2tJ2laVZ0VwwwIkViwejh0oru9Xs9mu2o1+j13H6I2eyyv7OK87SvK06I6NIyVi6sjo60Wj1pWvJfM28/s6/7s0P7l4r+iLT1PHq7FTbv8fxGEBpVjaVUoj6Wa4VZGRldbLzn9Nc1/bHLfJ1BRXR5/Hxp5u8W+cobqmo+ws6X2LUcYpt02i+n0ul0C4pGEMrq62FcqKZTHDr0SKQ2jcfC/maN465bLRaBw3pcBBUqUwhgwcWFDQ0x8fCAiKgCbPunijyZNpUnVXVgQQRgKlcTBjBxQWFDQ0L438GK82TBnCOZ8N5ZPJFAMUg4CpXFKlcYMHFloaM5csSS6vOk6BmbxH5Y2PeZGZmHgpBBXFKFSpwjW6vc+VVahdmZmZi3dKystWpa3hfztxnksmi0TqdJgwYuLilDrt33LnNtRlWo7uzMxZmLB52S76m2oLeM/K/i39GypCsX1emGDFymo1nNddzEsP57q9HdmZmZiWJIcWexZVCBeC/oPiH7R2j9ML54PmRvJ2o5o+qQoVKkHKUejOzlyxYk4T29oFVVCBOm08l2DmW3b3te+7fusqTZGUghu1KvRqFyxYn2Sc9pkwioioE+ZnOnHeYbRv22bvs+9xqjqwYMC9mq1C/bv27eycmJrNUVUExMyebz49yTZt82zc9j32NkdWDBnq1C/bv39+/7ISEwioioJ/NpUlSfHd62bdds1+x7rKisGB//8QAQBEAAgEDAAQLBQQIBwAAAAAAAQIDAAQRBQYSIRATIDAxMkBBUWGRIlJxgaEUQmKxByNDU2OSwcIVRFBUcqOy/9oACAEDAQM/ADUr9CGrmQ9FX13giE7J+824Vaw7Jnk2z7qbh61bWgAgt1TzA3/M1jHaJd2UAqJcFzVnbYKW658SM0e1xe7SDoFAd1DgHascAPODnxwEcI5kVnsZH+rZonuqzfpgHqRVm/VZ0+ByPrUwyYpkfyPsmru3yZIGA8cZHqKI57PLxwAbycUo6q/M1I33j8t1Y5Fjd52ogre8m41cQ5aI8avl1vSipII5jHM5rAoCvCiTk8vz4La6zxkYz7w3GpY8mI8Yvo1FSQQQfA8vHMYGeDPLxk5rUnQZdb7WiwjdemNZRLIPike01fo7sywhl0heecFvgf8AcyVqqCdjV/ShHnxS/wB5rUiVgs+i9LQefFROv0kr9H2sDpHaazWyytuEVzm2cnwHGhQT8KguV9tFYdxpG3xS48mq8Toj2vNSDV0v+Wk/lNXH+3k/lNSjrRMPiDw45RODzGoWrvGIdL/bp1/Y2IE/q+Qn1rWG8Lx6F0Tb2Kd0sx+0S/0UehrWzWMt/imsF3cof2bSFYvlGuFHJzWtupzxJa37z2gO+0nYvHj8HenyrROuVtm2k4q6RczWkpHGJ5qfvL50jYzlaVgCCDwedRSdeNW/5AGrOXOEKHxU1MuTFIr+R9k1dQ9aBx5gZHqKIPBO+MRkeZ3UEwXOT2K90Zd293Z3LwTwsGjkQ4KmrfW+0+z3OxDpOBMyxjcsq/vE/qO6mU5BonrUD0Hkg9IBoDoGOy3mir22vbOdoZ4HDxuvSCKtNcNErOuzHdw4W6gH3G95fwt3cJ7Xf6raXttIWbb09mSMnCyxnrI1aP1k0Va6RsZdqKUb1PWjcdKMO4jkEdjxyBw3WpeldshpbGchbqAeHc6fjWrPStlbXtncpNbzoHjkToYHhBorvHRzcFsm3LKEHn300mVt02R77dPyHPX2o17xMwe40XM+ZoB1oz+8iz97xHfWjNPaPt7/AEdeJc20wykiH1BHSCO8HhzWBtr0d45i0slzPOq+C9LH5CpHytrFsD333n0qWdy8sjOx72Oed3UTRNawaj3xuNG3OYnI4+2ky0MwHiO4+DDfWqut6wwG4FhftgG1uGA2m/hPuD/nw5FbByvVP05EcK7UkiovixAFaPgyIy0zfh3L6mr+5yEYQr4J0+tFiWYkk9JPPY4Mnka8arLHFb6Wa5tk3C3u/wBcgA7lJ9pR5A1a3OwmkdBbEne0Ev5K4/rWqVwFyt7GfBokP/lzWqMqECe4Of4VaCJxFHcv8VVf7q4wfqbJR5u+foBWlJ+iYRjwjXH1OTUkzbUkjO3ixJPabqyZQWLp9RUc6qRJSuB7VPHj2sio51BDb6B7VJA4eNipFHKpIdlvzoOF9qipUq1JOACcNWe1EbxUkTrHI3wNbQX2qIKkNQlAVjvrIG/tWDkU6kIx3iidmmBUg0ZVAPJ//9k="/>
```

Comment:
actually this is not svg, this is png image encoded to base64 inside of svg circle
not critical, just for you to know
piu piu copy from figma magic :)